ProcessMaker/Http/Controllers/Api/ProcessController.php

```diff
@@ -1711,8 +1717,23 @@ public function saveImagesIntoMedia(Request $request, Process $process)
             foreach ($request->imagesCarousel as $image) {
                 if (is_string($image['url']) && !empty($image['url'])) {
                     if (!$process->media()->where('collection_name', 'images_carousel')
-                        ->where('uuid', $image['uuid'])->exists()) {
-                        $process->addMediaFromBase64($image['url'])->toMediaCollection('images_carousel');
+                    ->where('uuid', $image['uuid'])->exists()) {
+                        if ($image['type'] === self::CAROUSEL_FILE_TYPES['IMAGE']) {
+                            $process
+                            ->addMediaFromBase64($image['url'])
+                            ->withCustomProperties(['type' => $image['type']])
+                            ->toMediaCollection('images_carousel');
+                        }
+                        if ($image['type'] === self::CAROUSEL_FILE_TYPES['EMBED']) {
+                            $fakeFile = File::image('photo.jpg');
```

Comment:
Please upload a image in the `resources/img/launchpad-images/embed.png `
Use this image in this section download the image icon related to embed